Design2/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Business Blog Card</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <div class="card-container">
+      <div class="card">
+        <div class="image-container">
+          <img src="picture.jpg" alt="" class="image" />
+        </div>
+        <div class="body">
+          <h2 class="title">Perfect solution for small business</h2>
+          <p class="description">
+            Small businesses need to generate leads to grow. You can use tools
+            like Ringy.
+          </p>
+          <div class="media">
+            <img
+              src="avatar.png"
+              class="imageM"
+            />
+            <div class="bodyM">
```

Comment:
what is this `bodyM, titleM, positionM`

Reply:
it is for the media section ...

Reply:
then it should be mediaImage, and mediaBody